frontend/src/metabase/search/analytics.ts

```diff
@@ -0,0 +1,38 @@
+import { trackSchemaEvent } from "metabase/lib/analytics";
+import type {
+  SearchQueryParamValue,
+  URLSearchFilterQueryParams,
+} from "metabase/search/types";
+
+function getContentType(
+  type: SearchQueryParamValue,
+): Array<SearchQueryParamValue> | null {
+  if (type) {
+    return Array.isArray(type) ? type : [type];
+  }
+  return null;
+}
+
+export const trackSearchEvents = ({
```

Comment:
nit: Why pass an object here with just one parameter?

Reply:
Ah I did that out of habit. I'll remove it here